components/rmrk/Gallery/History.vue

```diff
@@ -377,7 +377,8 @@ export default class History extends mixins(
       event['Block'] = String(newEvent['blockNumber'])
 
       // ID for b-table: Use a unique key of your data Object for each row.
-      event['ID'] = newEvent['timestamp'] + newEvent['id']
+      event['ID'] =
```

Comment:
In Squid Event id is
`export const eventId = (id: string, event: Action) => `${id}-${event}${nanoid()}``